codebase2/util-term/U/Util/Type.hs

```diff
@@ -61,7 +61,7 @@ flattenEffects es = [es]
 generalize :: (Ord v) => [v] -> TypeR r v -> TypeR r v
 generalize vs t = foldr f t vs
   where
-    f v t = if Set.member v (ABT.freeVars t) then forall v t else t
```

Comment:
Just a warning, not strictly necessary:
```
U/Util/Type.hs:84:1: warning: [GHC-64088] [-Wforall-identifier]
    The use of ‘forall’ as an identifier
    will become an error in a future GHC release.
    Suggested fix:
      Consider using another name, such as
      ‘forAll’, ‘for_all’, or ‘forall_’.
   |
84 | forall v body = ABT.tm () (Forall (ABT.abs () v body))
   | ^^^^^^
```

Reply:
Our build requires we have no warnings, so this change would require a rename at some point.